diff -Nur yquake2-QUAKE2_7_43/Makefile new/Makefile
--- yquake2-QUAKE2_7_43/Makefile	2020-02-24 16:15:39.000000000 +0000
+++ new/Makefile	2020-07-01 02:18:04.397380388 +0000
@@ -136,7 +136,7 @@
 		  -Wall -pipe -g -fwrapv
 CFLAGS += $(OSX_ARCH)
 else
-CFLAGS := -std=gnu99 -O2 -fno-strict-aliasing \
+CFLAGS := $(CFLAGS) -std=gnu99 -O2 -fno-strict-aliasing \
 		  -Wall -pipe -g -ggdb -MMD -fwrapv
 endif
 
